containers/release-candidate/daffodil-release-candidate

```diff
@@ -281,6 +281,7 @@ case $PROJECT_REPO in
 
     echo "Building Convenience Binaries..."
     mkdir -p $DAFFODIL_RELEASE_DIR/bin/
+    yarn install
```

Comment:
@Shanedell, this yarn install seems to be required now, is this documented anywhere?

Reply:
Please do. Thanks.

Reply:
Did this also resolved that issue you had above?

Reply:
No, i still get the error with yarn install. I found some references that said it has to do with webpack 4 and newer versions of Node.js. So maybe your PR that updates the webpack version will fix it. I'll try a dry run from that branch and see if it fixes it.

Reply:
Using your branch doesn't fix it. I found this active thread which seems like it's the issue:

https://togithub.com/webpack/webpack/issues/14532

According to that it's an issue with webpack and Node.js 17, but this update is to Fedora 36 which only includes Node.js 16. There are some comments that mention F36 not working, but no solution. Setting NODE_OPTIONS doesn't work for me in the VM, with the error:

```
node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```

Seems something else is going on...

Reply:
@stevedlawrence Were you running a `yarn build` when you got the error? If so if is definetly something weird but if you tried running that `webpack` command on its own it may cause an error if you didn't build the source code yet.

Reply:
This was when running `yarn build`.

Reply:
Yeah that is something strange I can try to dig into it today based off of this branch

Reply:
@Shanedell 
I found steps that seems to fix this:
1. Update webpack to the very latest (5.73.0) as well as webpack-cli (4.10.0)
2. Set output.hashFunction to "sha512" in `build/extension.webpack.config.js`

Any reason why you didn't update webpack to 5.73.0 in your dependabot updates? It *think* we need at least something like 5.60 or something for the hashFunction change to work where we need it.

Reply:
I think this is the commit in webpack that makes it so `output.hashFunction` is used everywhere:

https://github.com/webpack/webpack/commit/4a8bf4c084f98bef8b47f0050c352f246ffb3527

That was merged in 5.54.0, so we at least need that version for things to work.